dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -241,7 +243,7 @@ std::wstring Microsoft::Windows::AppNotifications::Helpers::GetDisplayNameBasedO
     return displayName;
 }
 
-winrt::guid Microsoft::Windows::AppNotifications::Helpers::RegisterComActivatorGuidAndAssets()
+std::wstring Microsoft::Windows::AppNotifications::Helpers::GetOrCreateComActivatorGuid()
```

Comment:
This returns an r-value. So we should be using std::move constructs in the caller to avoid unecessary copies.

Reply:
Actually, we should probably leave this to the compiler. Based on what I am reading here: https://en.cppreference.com/w/cpp/language/copy_elision - the compiler tries to do a return value optimization first i.e.-"objects are constructed directly into the storage where they would otherwise be copied/moved to." - if it fails then it would switch to move and then finally to copy. Trying to std::move would end up counterintuitive to what is being achieved. I would be avoiding this in the PR. Let me know if you otherwise. Thanks.